### ISSUE TITLE
Could not read the QuickRoute file

### ISSUE THREAD

**Issue by Kiziev** [mikoro/orientview#3]
So why? What is the problem?

**Comment by lerppu** [comment]
Hey, Now that I'm trying to process my Jukola track, it seems like too long session ends with this kind of error. When I split my 1h 48 min gpx in half before Quickroute, it works fine. Dunno if it is issue with Orientview or Quickroute. I've had issues with Quickroute when the image is too big, so I'm guessing the issue could be with Quickroute

More specifically the error in log is 
"Could not find projection origin"
"Could not initialize QuickRoute reader"

**Comment by lerppu** [comment]
I guess neither Orientview nor Quickroute are developed anymore so I guess we'll just have to live with this unless someone wants to try fix it

**Comment by mikoro** [comment]
Hey @Kiziev and @lerppu! Can you upload somewhere both a working and a non-working Quickroute file so I can take a look at it?

**Comment by Kiziev** [comment]
@mikoro Thanks! I'm glad you're alive! :)
You can see the files here: [https://mega.nz/folder/9go0hCiB#eOzBjf3KevOwLx8_GviTcg](url). For one file does not work. Need to separate the two files and everything will work. But it's a waste of time, two edited videos. The problem is when more than about 25 points (CP) or more than 1h30min. I hope you understand what I'm writing.

**Comment by tomhula** [comment]
Hey there, I managed to fix the issue in [my fork](https://github.com/tomhula/orientview), but I have not made a release. I can easily publish a .deb package, but I have problems with the Windows release. If anyone would still be interested, write me a message, either here, or make an issue on the fork and I would make an effort to make the release. Or I can submit a pull-request to this repo, but I am not sure how if that is desirable @mikoro. 
@Kiziev

**Comment by mikoro** [comment]
@tomhula Thanks for the effort! I'll add you as a contributor to the repo so you can push the fixes straight to the master at your discretion. If in any way possible, it would be super nice to have the windows exe updated too 😄

**Comment by tomhula** [comment]
Yes, without the exe I find it as a kind of wasted effort. The issue is I have totally 0 experience with any C++ (or other native) languages and their linking, packaging etc. It was pretty difficult for me to even build the Linux executable. However I will look into it, because I think this is the best software for orienteering headcam footage (there overall aren't even many alternatives) Maybe do you think you would be able to assist a little?  Rather some questions about stuff I don't understand?

**Comment by mikoro** [comment]
@tomhula Happy to do it. Just send me email to schedule a remote meeting.

**Comment by alwfree** [comment]
Is it would be a new version of setup file for windows?

**Comment by tomhula** [comment]
@alwfree You can try the latest build, which you can download [here](http://files.tomashula.cz/orientview-commit-ec63d53-windows-binaries.zip) (I wasn't able to upload them here). If @mikoro will not reply in few days, I will push the official release with the builds. Anyway let us know how it works please, or if you encounter any issues (except those already reported or in known issues in readme) create a new issue. Thanks.

**Comment by alwfree** [comment]
Solution with splitting in 1h48m helps me to work. Thanks)

**Comment by tomhula** [comment]
Fixed in [v1.2.0](https://github.com/mikoro/orientview/releases/tag/v1.2.0)
